src/Storage/S3/ExasolExportAdapter.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Db\ImportExport\Storage\S3;
+
+use Doctrine\DBAL\Connection;
+use Keboola\Db\ImportExport\Backend\BackendExportAdapterInterface;
+use Keboola\Db\ImportExport\ExportOptionsInterface;
+use Keboola\Db\ImportExport\Storage;
+use Keboola\TableBackendUtils\Escaping\Exasol\ExasolQuote;
+
+class ExasolExportAdapter implements BackendExportAdapterInterface
+{
+    /** @var Connection */
+    private $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public static function isSupported(Storage\SourceInterface $source, Storage\DestinationInterface $destination): bool
+    {
+        if (!$source instanceof Storage\SqlSourceInterface) {
```

Comment:
prislo by mi citelnejsi `return ($source instanceof Storage\SqlSourceInterface && $destination instanceof Storage\S3\DestinationFile)` ale asi muze byt... nebo to je zase nachystane na dalsi implementace...

Reply:
To ti přijde čitelnější? :O z toho sa vaří mozek :D